azure-quantum/tests/unit/test_microsoft_qc.py

```diff
@@ -231,6 +255,22 @@ def test_estimator_params_validation_missing_fields(self):
                                        "set"):
             params.as_dict()
 
+    def test_estimator_params_validation_measurement_error_rates_valid(self):
+        params = MicrosoftEstimatorParams()
+
+        params.error_budget = 0.1
+        params.qubit_params.name = QubitParams.GATE_NS_E3
+        params.qubit_params.instruction_set = "gate_based"
+        params.qubit_params.t_gate_error_rate = 0.03
+        params.qubit_params.t_gate_time = "10 ns"
+        params.qubit_params.idle_error_rate = 0.02
+        params.one_qubit_measurement_error_rate = 0.01
+        params.qubit_params.two_qubit_joint_measurement_error_rate = \
+        MeasurementErrorRate(process = 0.02, readout = 0.03)
+
+        params.as_dict()
```

Comment:
You should compare this to an expected value. You will notice that the `MeasurementErrorRate` is not correctly serialized.